components/component-stylesheets/button.css

```diff
@@ -0,0 +1,162 @@
+.btn{
+    padding: 1rem 2rem ;
+    font-size: 1.3rem;
+    background-color: var(--mr-default-color); /*secondary color*/
+    border: none;
+    text-align: center;
+    outline: none;
+    cursor: pointer;
+    margin: 1rem 0rem;
+    color: var(--mr-white-text-color);
+    font-weight: 500;
+    border-radius: 0.5rem;
+}
+
+.btn:hover{
+    background-color: var(--mr-dark-default); /* dark secondary*/
+}
+
+/* primary buttons styles */
+
+/* solid primary button */
+.btn-primary{
+    background-color: var(--mr-primary-color);
+}
+
+.btn-primary:hover{
+    background-color: var(--mr-dark-primary); /*dark primary*/
+}
+
+/* primary outline button */
+.btn-primary-outline{
+    background-color: var(--mr-white-text-color);
+    border:1px solid var(--mr-primary-color);
+    color: var(--mr-primary-color);
+}
+
+.btn-primary-outline:hover{
+    background-color: var(--mr-dark-primary);
+    color: var(--mr-white-text-color);
+}
+
+.btn-link{
+    color: var(--mr-default-color);
+    background-color: transparent;
+    border: none;
+    cursor: pointer;
+    padding: 0;
+    font-size: 1.3rem;
+}
+
+/* link button */
+.btn-link-primary{
+    text-decoration: underline;
+    text-underline-position: under;
+    text-decoration-color: var(--mr-primary-color);
+}
+
+.btn-link-primary:hover{
+    color: var(--mr-primary-color);
+}
+
+    /* icon buttons */
+.btn-icon{
+    background: none;
+    border: none;
+    font-size: 1.8rem;
+    color: var(--mr-primary-color);
+}
+
+.btn-icon:hover{
+    color: var(--mr-dark-primary);
+}
+
+/* text-icon */
+.btn-text-icon{
+    background-color: var(--mr-primary-color);
+}
+
+.btn-text-icon:hover{
+    background-color: var(--mr-dark-primary);
+}
+
+.btn-text-icon .btn-icon{
+    color: var(--mr-white-text-color);
+    font-size: 1.3rem;
+}
+
+/* secondary button styles */
+.btn-secondary{
+    background-color:var(--mr-secondary-color);
+    color: var(--mr-light-text-color);
+}
+
+.btn-secondary:hover{
+    background-color: var(--mr-default-color);
+}
+
+/* secondary outline button */
+.btn-secondary-outline{
+    background-color: var(--mr-white-text-color);
+    border:1px solid var(--mr-secondary-color);
+    color: var(--mr-light-text-color);
+}
+
+.btn-secondary-outline:hover{
+    background-color:var(--mr-secondary-color);
+}
+
+/* secondary link button */
+.btn-link-secondary{
+    text-decoration: underline;
+    text-underline-position: under;
+    text-decoration-color: var(--mr-secondary-color);
+}
+
+.btn-link-secondary:hover{
+    color: var(--mr-dark-secondary);
+}
+
+/* icon button secondary */
+.btn-icon-secondary{
+    background: none;
+    border: none;
+    font-size: 1.8rem;
```

Comment:
change color only in this class and you can use btn-icon for background border and font-size.

Reply:
thanks for pointing out. fixed in recent commit